components/top-bar/index.tsx

```diff
@@ -219,11 +219,7 @@ const TopBar = () => {
         </div>
         <MarketSearch />
         <div className="center relative ml-auto gap-3">
-          {process.env.NEXT_PUBLIC_SHOW_AIRDROP !== "true" ? (
-            <GetTokensButton />
-          ) : (
-            <AirdropButton />
-          )}
+          <GetTokensButton />
```

Comment:
Looks good. You could think about removing all related components (including claim.tsx) for the Airdrop button, since we wouldn't use it anymore.

Reply:
In case we decide to do another airdrop for whatever reason I'll leave it  for now